app/src/main/res/xml/tools_settings.xml

```diff
@@ -50,6 +50,22 @@
             android:title="@string/preference_tts_force_english_title"
             app:iconSpaceReserved="false" />
 
+        <SwitchPreferenceCompat
```

Comment:
I think we can just have a single `summary` that explains what the setting does when it is switched on. No need for the `summaryOff`/`summaryOn`.

Reply:
Fixed in [8ba2f1a](https://github.com/flex3r/DankChat/pull/309/commits/8ba2f1a93bf204ed3bddbffa668b5e8c446f767c)